content/developer/reference/backend/testing.rst

```diff
@@ -730,14 +731,113 @@ To start a tour from a python test, make the class inherit from
 
    def test_your_test(self):
        # Optional Setup
-       self.start_tour("/web", 'your_module.your_tour_name', login="admin")
+       self.start_tour("/web", "your_tour_name", login="admin")
        # Optional verifications
 
+Writing an onboarding tour
+--------------------------
+
+The main purpose of the onboarding tours is to guide the user. They can be started with the
```

Comment:
Considering that you have a "Running onboardng tours" section, I would remove "The main purpose of the onboarding tours is to guide the user." completely and move the rest in that section.